RecoEgamma/ElectronIdentification/src/SoftElectronMVAEstimator.cc

```diff
@@ -3,21 +3,9 @@
 #include "DataFormats/TrackReco/interface/TrackFwd.h"
 #include "DataFormats/GsfTrackReco/interface/GsfTrack.h"
 #include "DataFormats/GsfTrackReco/interface/GsfTrackFwd.h"
-#include "FWCore/ParameterSet/interface/FileInPath.h"
-#include "TMVA/Reader.h"
-#include "TMVA/MethodBDT.h"
-
-SoftElectronMVAEstimator::SoftElectronMVAEstimator(const Configuration & cfg):cfg_(cfg){
-  std::vector<std::string> weightsfiles;
-  std::string path_mvaWeightFileEleID;
-  for(unsigned ifile=0 ; ifile < cfg_.vweightsfiles.size() ; ++ifile) {
-    path_mvaWeightFileEleID = edm::FileInPath ( cfg_.vweightsfiles[ifile].c_str() ).fullPath();
-    weightsfiles.push_back(path_mvaWeightFileEleID);
-  }
-  
-  //initialize
-  //Define expected number of bins
 
+SoftElectronMVAEstimator::SoftElectronMVAEstimator(const Configuration & cfg):cfg_(cfg)
+{
   //Check number of weight files given
   if (ExpectedNBins != cfg_.vweightsfiles.size() ) {
     std::cout << "Error: Expected Number of bins = " << ExpectedNBins << " does not equal to weightsfiles.size() = "
```

Comment:
Replace with a LogError

Reply:
Sure I did that, and thanks for testing!

Reply:
I also removed this `assert(ExpectedNBins == cfg_.vweightsfiles.size())` in the `if(ExpectedNBins != cfg_.vweightsfiles.size() )` block. I guess that was redundant...